core/src/test/resources/sql/test.iq

```diff
@@ -0,0 +1,63 @@
+# test.iq - Join query tests
```

Comment:
you should probably call this file something else, perhaps planner.iq?
the comment is no longer accurate

Reply:
Good point! I didn’t think that far ahead when writing the demo.